### ISSUE TITLE
lancat -s not working

### ISSUE THREAD

**Issue by LuisGP** [lemunozm/lancat#1]
```bash
➜  11:05:28 ~ lancat -s
error: The argument '--search' cannot be used with '--name <user name>'

USAGE:
    lancat --discovery-ip <ip> --discovery-port <number> --listen --name <user name> --search --service-port <number>

For more information try --help

```

Ubuntu 18.04.2 LTS

**Comment by lemunozm** [comment]
Hi @LuisGP, there was a problem with the *cli* already fixed at v0.2.0

**Comment by LuisGP** [comment]
Thank you @lemunozm 
It's working now :)
